crates/swc_ecma_ast/src/ident.rs

```diff
@@ -254,6 +254,11 @@ impl Ident {
 
         Err(buf)
     }
+
+    #[inline]
+    pub fn is_dummy(&self) -> bool {
+        self.sym == *"" && self.span.is_dummy()
```

Comment:
Can you change the symbol comparison to `self.sym == js_word!("")`?

Reply:
Comparison of `JsWord` with `js_word!("")` is a comparison of `u32` to a constant